Nef_S2/include/CGAL/Nef_S2/Sphere_direction.h

```diff
@@ -95,18 +95,20 @@ bool strictly_ordered_ccw_at(const Sphere_point<R>& p,
   const Sphere_direction<R>& d2,
   const Sphere_direction<R>& d3)
 { CGAL_assertion(d1.has_on(p) && d2.has_on(p) && d3.has_on(p));
-  typename R::Point_3 p0(0,0,0);
-  typename R::Point_3 p1(CGAL::ORIGIN + d1.orthogonal_vector());
-  typename R::Point_3 p2(CGAL::ORIGIN + d2.orthogonal_vector());
-  typename R::Point_3 p3(CGAL::ORIGIN + d3.orthogonal_vector());
+  typedef typename R::Vector_3 Vector_3;
+  Vector_3 v0 = p - CGAL::ORIGIN;
```

Comment:
Might there be any advantage to making `v0` through to `v3` declared as  `const Vector_3&` even if Vector_3 is cheap to copy?